packages/client/src/forms/user/fieldDefinitions/createUser.ts

```diff
@@ -79,38 +78,38 @@ export function userSectionFormType(): ISerializedFormSection {
         title: userFormMessages.userDetails,
         fields: [
           {
-            name: 'firstNamesEng',
+            name: 'familyName',
             type: TEXT,
-            label: userFormMessages.firstNameEn,
+            label: userFormMessages.lastName,
             required: true,
             initialValue: '',
             validator: [{ operation: 'englishOnlyNameFormat' }],
             mapping: {
               mutation: {
                 operation: 'fieldToNameTransformer',
-                parameters: ['en', 'firstNames']
+                parameters: ['en', 'familyName']
               },
               query: {
                 operation: 'nameToFieldTransformer',
-                parameters: ['en', 'firstNames']
+                parameters: ['en', 'familyName']
               }
             }
           },
           {
-            name: 'familyNameEng',
+            name: 'firstNamesEng',
```

Comment:
any particular reason why one's called "familyName" while the other "firstNamesEng"? :sweat_smile:

Reply:
Thanks @Zangetsu101, good question. I may have misunderstood @rikukissa requested change from above. How I understood it was that a person can have multiple names from different languages, is this interpretation correct?

Reply:
@Siyasanga Sorry, I should have been clearer! A user can only have one name, always in English. The `-Eng` suffix is technical debt from the past :) If we need to have multiple names in different scripts later on we can build a new solution. So for now, all `En` and `Eng` suffixes should be removed both from core and country config client.csv